docs/features/nullable-reference-types.md

```diff
@@ -67,13 +67,9 @@ If there is no `[NonNullTypes]` attribute at any containing scope, including the
 [NonNullTypes(false), Nullable(new[] { false, true })] string[] ObliviousMaybeNull; // string?[]~
 [NonNullTypes(true), Nullable(new[] { false, true })] string[] NotNullMaybeNull; // string?[]!
 ```
-`NonNullTypesAttribute` is not synthesized by the compiler. If the attribute is used explicitly in source, the type declaration must be provided explicitly to the compilation. The type should be defined in the framework.
+The `NonNullTypesAttribute` is always implicitly included in a compilation, but is only emitted if it is referenced in source. It cannot be defined explicitly in source or referenced from metadata.
```

Comment:
>It cannot be defined explicitly in source or referenced from metadata. [](start = 127, length = 70)

Even in a .Net module?

Reply:
I'll update the comment. The attribute can be defined in source after all.
But it cannot be referenced from an assembly (the injected symbol will be used instead) or added from a .Net module (collision).